api/src/test/java/org/openmrs/module/fhir/api/PersonServiceTest.java

```diff
@@ -126,71 +126,6 @@ public void updatePerson_shouldGenerateOmsPerson() throws Exception {
 		assertNotNull(retrievedPerson);
 	}
 
-	/**
-	 * @verifies update Person
-	 */
-	@Test
```

Comment:
I think we should have such a test, maybe it could be improved, but something like that should be verified.

Reply:
Test added